visualizations/store-map-viz/queries/index.tsx

```diff
@@ -1,43 +1,34 @@
 import { timeRangeToNrql } from "../utils";
 
-// const lookupTableQuery = () => {
-//   return `FROM lookup(Stores) SELECT *`;
-// };
+const markerNRQL = (markersQuery,timeRange) => {
 
-// export const lookupTableNrql = () => {
-//   return `nrql( query: "${lookupTableQuery()}" ) { results }`;
-// };
+  if(markersQuery.match(/since|until/gi)) {
+    return markersQuery.replace(/(\r\n|\n|\r)/gm," ");
+  } else {
+    // Generate the time range part of the NRQL query
+    const timeRangePart = timeRangeToNrql(timeRange);
 
-// export const nerdGraphLookupQuery = () => `
-//     query($id: Int!) {
-//       actor {
-//         account(id: $id) {
-//           lookup: ${lookupTableNrql()}
-//         }
-//       }
-//     }
-// `;
+    // Construct the full NRQL query, remove line breaks
+    let query = `${markersQuery.replace(/(\r\n|\n|\r)/gm," ")} ${timeRangePart}`;
+    console.log("OVERFETCH!",Date.now(), query); //TODO: Mat to look at overfetch on window resize
```

Comment:
@matewilk Can you look into this overfetch issue when resizing window?

Reply:
I'll look into it